src/abstractdataframe/selection.jl

```diff
@@ -412,7 +416,8 @@ end
 
 function _fix_existing_columns_for_vector(newdf::DataFrame, df::AbstractDataFrame,
                                           allow_resizing_newdf::Ref{Bool}, lr::Int,
-                                          @nospecialize(fun))
+                                          wfun::Ref{Any})
+    fun = only(wfun)
```

Comment:
It just occurred to me that this syntax works: `(fun,)::Ref`. That avoids adding one line for each argument and playing with argument names. The `Any` doesn't add anything AFAICT.

Reply:
I want to use `Ref{Any}` to ensure we have exactly one method instance generated (same comment as in several places above). This is mostly for the future development.

Regarding `(fun,)::Ref{And}` this is smart indeed. Do you feel it will be more readable than `fun = only(wfun)` (I wanted to keep consistent style as sometimes we just pass through `wfun` without unwrapping). But I am OK with both - just wondering which method is better.

Reply:
I find it a bit more readable, especially since it allows keeping `fun` as the name everywhere.

Reply:
OK - I will change then. But when `Ref{Any}` is only passed around and not accessed I will keep `wfun` to avoid unwrap-rewrap operation.